cloud-language-starter/src/main/java/com/sample/autoconfig/LanguageAutoConfig.java

```diff
@@ -94,10 +70,19 @@ public TransportChannelProvider defaultLanguageTransportChannelProvider() {
     return LanguageServiceSettings.defaultTransportChannelProvider();
   }
 
+  // bean for overriding retry settings on service-level
+  @Bean
+  @ConditionalOnMissingBean(name = "languageServiceRetry")
+  public Retry languageServiceRetry() {
+    return this.clientProperties.getRetry();
+  }
+
   @Bean
   @ConditionalOnMissingBean
-  public LanguageServiceClient languageServiceClient(@Qualifier("languageServiceCredentials") CredentialsProvider credentialsProvider,
-      @Qualifier("defaultLanguageTransportChannelProvider") TransportChannelProvider defaultTransportChannelProvider)
+  public LanguageServiceSettings languageServiceSettings(
+          @Qualifier("languageServiceCredentials") CredentialsProvider credentialsProvider,
+          @Qualifier("defaultLanguageTransportChannelProvider") TransportChannelProvider defaultTransportChannelProvider,
+          @Qualifier("languageServiceRetry") Retry languageServiceRetry)
```

Comment:
This would be `RetrySettings` from GAPIC rather than the `Retry`.

Reply:
Updated per comment above